lib/matplotlib/table.py

```diff
@@ -453,19 +476,28 @@ def _auto_set_font_size(self, renderer):
 
         if len(self._cells) == 0:
             return
-        fontsize = list(six.itervalues(self._cells))[0].get_fontsize()
-        cells = []
+
+        fontsize = None
+        grow = self._bbox
```

Comment:
better to rely on truthiness of self._bbox than to have it break if someone calls it with bbox = False?